package.json

```diff
@@ -248,5 +252,21 @@
   "bugs": {
     "url": "https://github.com/Trustroots/trustroots/issues"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "husky": {
```

Comment:
@simison What do you think about moving `husky` and `lint-staged` config to `husky.config.js` and `lint-staged.config.js` respectively (or rc files) to keep package.json uncluttered?

If you think it's a good idea and if you don't have time for it, I will do it.

Reply:
Sure, I generally like that approach for configs as well.

Reason why I put them here initially was that they're so tied to actual `package.json` scripts. Configs themself were very "thin" — i.e. there wasn't much more to them than what commands to run — that having the `scripts` and these in the same file felt like easier to read.

Similar with `browserslist` config:

https://github.com/Trustroots/trustroots/blob/d07f7b66cebb4747d2a0a3ea7eac1749d7a98c59/package.json#L16-L18

But little by little we're having more of these in the `package.json` so it makes totally sense to move them to separate files. 👍  It also makes them easier to discover.

Reply:
Makes sense. Then we can as well keep it in `package.json` and move it later. :slightly_smiling_face: